docs/customservices.md

```diff
@@ -272,4 +272,9 @@ Configuration example:
   warning_value: 50
   danger_value: 80
   api_token: "PVEAPIToken=root@pam!your-api-token-name=your-api-token-key"
+  # values below this line are optional (default value are false/empty):
+  hide_decimals: true # removes decimals from stats values.
+  hide: ["vms", "vms_total", "lxcs", "lxcs_total", "disk", "mem", "cpu"] # hides values included in the array
+  small_font_on_small_screens: true # uses small font on small screens (like mobile)
```

Comment:
I think this one should be automatic using CSS & [bulma responsive classes](https://bulma.io/documentation/overview/responsiveness/#breakpoints).

Reply:
I think most users want to show as much info as possible, that's why I thought it was better to put what you want to hide. Following that line, you don't have to add an extra config line if you want to show all info.

Anyhow, I can change the implementation if your think it's better to use show instead of hide. In that case, I would change everything as I don't like to mix show and hide fields. Your choice :)

Reply:
Sorry, I wrote this for desktop version frist (see below on your other comment) but thing is easier here. I can set smaller font automatically if a field which is not a total is hidden.

Anyhow, as there are so many screen sizes on mobile... I can set it by default and let and `avoid` field in case someone needs it.

Do you feel better about that option?  I can apply the same to desktop.